### PR TITLE
feature: pass multiple group ids on chunk creation

### PR DESCRIPTION
**Pull request by skeptrunedev**

### CODE REVIEW

File: server/src/handlers/chunk_handler.rs [hunk 1]
```diff
@@ -47,8 +47,8 @@ pub struct CreateChunkData {
     pub chunk_vector: Option<Vec<f32>>,
     /// Tracking_id is a string which can be used to identify a chunk. This is useful for when you are coordinating with an external system and want to use the tracking_id to identify the chunk.
     pub tracking_id: Option<String>,
-    /// Group is the id of the group that the chunk should be placed into. This is useful for when you want to create a chunk and add it to a group in one request.
-    pub group_id: Option<uuid::Uuid>,
+    /// Group ids are the ids of the groups that the chunk should be placed into. This is useful for when you want to create a chunk and add it to a group or multiple groups in one request. Necessary because this route queues the chunk for ingestion and the chunk may not exist yet immediatley after response.
+    pub group_ids: Option<Vec<uuid::Uuid>>,
```
Comment (fastfists) [inline server/src/handlers/chunk_handler.rs:51]: Does utopia auto update?

**Review by fastfists — COMMENTED** [review]
Left a comment

**Review by densumesh — APPROVED** [review]
LGTM

File: server/src/handlers/chunk_handler.rs (hunk 1 above)
Comment (skeptrunedev) [reply server/src/handlers/chunk_handler.rs:51]: no, I wrote this

Comment (fastfists) [reply server/src/handlers/chunk_handler.rs:51]: No will utopia update the json parameters based on this

Comment (densumesh) [reply server/src/handlers/chunk_handler.rs:51]: Yeah it does

**Pull request merged**